seventh-lesson/src/task25/ATM.java

```diff
@@ -0,0 +1,122 @@
+package task25;
+
+public class ATM {
+    int numberOfBill20;
+    int numberOfBill50;
+    int numberOfBill100;
+
+    public ATM(int numberOfBill20, int numberOfBill50, int numberOfBill100) {
+        this.numberOfBill20 = numberOfBill20;
+        this.numberOfBill50 = numberOfBill50;
+        this.numberOfBill100 = numberOfBill100;
+    }
+
+    @Override
+    public String toString() {
+        return "ATM{" +
+                "numberOfBill20=" + numberOfBill20 +
+                ", numberOfBill50=" + numberOfBill50 +
+                ", numberOfBill100=" + numberOfBill100 +
+                '}';
+    }
+
+    public void addBill20(int quantity20) {
+        this.numberOfBill20 += quantity20;
+        System.out.println("Added " + quantity20 + " bills of 20");
+        System.out.println("Total at ATM " + this.numberOfBill20 + " bills of 20");
+        System.out.println();
+    }
+
+    public void addBill50(int quantity50) {
+        this.numberOfBill50 += quantity50;
+        System.out.println("Added " + quantity50 + " bills of 50");
+        System.out.println("Total at ATM " + this.numberOfBill50 + " bills of 50");
+        System.out.println();
+    }
+
+    public void addBill100(int quantity100) {
+        this.numberOfBill100 += quantity100;
+        System.out.println("Added " + quantity100 + " bills of 100");
+        System.out.println("Total at ATM " + this.numberOfBill100 + " bills of 100");
+        System.out.println();
+    }
+
+    public boolean getCash(int sum) {
```

Comment:
Because the task states: "..Сделать функцию 
снимающую деньги. На вход передается сумма денег. На выход – булевское 
значение (операция удалась или нет)...."